frontend/layouts/booking.tsx

```diff
@@ -1,4 +1,4 @@
-import { Footer } from '@/components/shared/Footer'
+import { BookingFooter } from '@/components/shared/footer/BookingFooter'
```

Comment:
You can create a barrel file inside the footer folder. The barrel file is index.ts, and what is essentially does is export everything in that folder so all the imports will be much simpler. Instead of importing from the exact file directory, you can just import from the @/components/shared/footer directory.